google-cloud-clients/google-cloud-firestore/src/main/java/com/google/cloud/firestore/FirestoreImpl.java

```diff
@@ -140,21 +140,31 @@ public DocumentReference document(@Nonnull String documentPath) {
   @Override
   public ApiFuture<List<DocumentSnapshot>> getAll(
       @Nonnull DocumentReference... documentReferences) {
-    return this.getAll(documentReferences, null, null);
+    return this.getAll(documentReferences, null, null, null);
```

Comment:
For cases like this, it might make sense to add inline comments to describe the arguments:

```
   return this.getAll(documentReferences, /* foo= */ null, /* bar= */ null, /* baz= */ null);
```